src/solvers/prop/prop_conv_solver.h

```diff
@@ -35,6 +35,9 @@ class prop_conv_solvert : public conflict_providert,
 
   virtual ~prop_conv_solvert() = default;
 
+  // non-iterative eager conversion
+  virtual void finish_eager_conversion();
```

Comment:
Just curious: this used to be `protected`. Why is making it `public` the right thing to do?

Reply:
Yes, I am thinking about the use of this for an eager conversion, say to Dimacs, or something else.
Right now, we trigger the "finishing" by calling `dec_solve`, which then doesn't actually do solving, but returns an error, which we then ignore. It would be clearer to feed the constraints to the decision procedure, and then call `finish_eager_conversion`, and then be done.